server/container_create.go

```diff
@@ -1237,14 +1227,32 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 
 	sort.Sort(orderedMounts(mounts))
 
+	config := specgen.Spec()
+	mountDestinations := map[string]int{}
+	for j, mount := range config.Mounts {
+		if i, ok := mountDestinations[mount.Destination]; ok {
+			return nil, fmt.Errorf("CRI-O configured duplicate mounts for %q (%d and %d)", mount.Destination, i, j)
```

Comment:
this changes the behavior to error out now, I don't think we want this

Reply:
> this changes the behavior to error out now, I don't think we want this

I don't think we should ever hit this, because it's for when *we* configure multiple defaults for the same destination.  Do you think we would do that accidentally?  Intentionally?  If it happened, how would you like `crio` to respond?  This error is because I'd only ever expect it happening accidentally, and I want to hear about it in our CI tests before we merge the buggy code.